### PR TITLE
Sort `PlaceResult` for NearbyStations usecase

### PR DESCRIPTION
**Pull request by vasile**

### CODE REVIEW

File: Sources/OJP/OJPHelpers.swift [hunk 1]
```diff
@@ -29,9 +29,20 @@ enum OJPHelpers {
     }
 
     class LocationInformationRequest {
-        public static func placeResultsSorted(from _: (long: Double, lat: Double), placeResults: [OJPv2.PlaceResult]) -> [OJPv2.PlaceResult] {
-            // TODO: implement the logic
-            placeResults
+        public static func placeResultsSorted(from point: (long: Double, lat: Double), placeResults: [OJPv2.PlaceResult]) -> [OJPv2.PlaceResult] {
+            var nearbyObjects: [NearbyObject<OJPv2.PlaceResult>] = []
+            for placeResult in placeResults {
```
Comment (r3to) [inline Sources/OJP/OJPHelpers.swift:34]: This could go into  `placeResults.map { }`

File: Sources/OJP/OJPHelpers.swift [hunk 2]
```diff
@@ -29,9 +29,20 @@ enum OJPHelpers {
     }
 
     class LocationInformationRequest {
-        public static func placeResultsSorted(from _: (long: Double, lat: Double), placeResults: [OJPv2.PlaceResult]) -> [OJPv2.PlaceResult] {
-            // TODO: implement the logic
-            placeResults
+        public static func placeResultsSorted(from point: (long: Double, lat: Double), placeResults: [OJPv2.PlaceResult]) -> [OJPv2.PlaceResult] {
```
Comment (r3to) [inline Sources/OJP/OJPHelpers.swift:32]: Maybe directly return `[NearbyObject<OJPv2.PlaceResult>]` so we could display the distance in the UI.

File: Sources/OJP/OJPHelpers.swift (hunk 1 above)
Comment (vasile) [reply Sources/OJP/OJPHelpers.swift]: Moved to `GeoHelpers`

File: Sources/OJP/OJPHelpers.swift (hunk 2 above)
Comment (vasile) [reply Sources/OJP/OJPHelpers.swift]: Good idea, implemented as such

**Comment by r3to** [comment]
lgtm

**Review by r3to — APPROVED** [review]

**Review by terencebls — APPROVED** [review]

**Pull request merged**